zellij-server/src/pty.rs

```diff
@@ -201,9 +201,13 @@ impl Pty {
         }
     }
     pub fn get_default_terminal(&self) -> TerminalAction {
+        let shell = PathBuf::from(env::var("SHELL").unwrap_or_else(|_| "/bin/sh".to_string()));
```

Comment:
I think we should add a log entry for this, something like "Cannot read 'SHELL' env, falling back to ...". This way we (and the user) can diagnose why zellij didn't pick up some shell configured by the user.

Reply:
Ok, I'll do that. Thanks.